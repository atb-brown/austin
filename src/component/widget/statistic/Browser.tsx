import { browserName } from "react-device-detect";
import React from "react";

/**
 * This component displays the user's browser.
 *
 * @return {ReactElement} React Component
 */
export default function Browser(): JSX.Element {
  const title = "Browser";
  return (
    <div style={{ display: "flex", flexDirection: "column", fontSize: "50%" }}>
      <div>{title}</div>
      <div>{browserName}</div>
    </div>
  );
}
