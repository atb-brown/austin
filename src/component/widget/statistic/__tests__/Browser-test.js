import Browser from "../Browser";
import { render } from "@testing-library/react";

it("renders to match snapshot", () => {
  const { asFragment } = render(<Browser />);

  expect(asFragment()).toMatchSnapshot();
});
