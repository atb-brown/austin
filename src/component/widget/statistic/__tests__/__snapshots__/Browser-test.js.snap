// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match snapshot 1`] = `
<DocumentFragment>
  <div
    style="display: flex; flex-direction: column; font-size: 50%;"
  >
    <div>
      Browser
    </div>
    <div>
      WebKit
    </div>
  </div>
</DocumentFragment>
`;
